Ginger/GingerCoreNET/Database/DatabaseOperations.cs

```diff
@@ -758,6 +780,26 @@ public List<object> FreeSQL(string SQL, int? timeout = null)
             return ReturnList;
         }
 
+        private bool IsConnectionDisposed()
+        {
+            bool isConnectionDisposed;
+            try
+            {
+                oConn.Open();
+                isConnectionDisposed = false;
+            }
+            catch (ObjectDisposedException)
+            {
+                isConnectionDisposed = true;
+            }
+            catch (InvalidOperationException)
```

Comment:
Replace InvalidOperationException with general exception so it will catch other exceptions too

Reply:
Exception types are used to check if the connection is closed or not. 
ObjectDisposedException = Connection Closed
InvalidOperationException = Connection Open
With general Exception we have no way of knowing the connection state. Do you think we should catch it or let it bubble up?